CONTRIBUTING.md

```diff
@@ -1,34 +1,115 @@
-# Contributing to scrumlr.io
+# Contributing
+Thank you for considering contributing to Scrumlr and taking the time to explore our contribution guide. We greatly appreciate your interest and willingness to help improve Scrumlr with your valuable contributions. Your involvement and contributions play a significant role in enhancing Scrumlr and making it even better. We warmly welcome pull requests from all contributors, big or small. For major changes, we kindly request to [open a discussion](https://github.com/inovex/scrumlr.io/discussions) first, allowing us to collaborate and discuss your ideas and proposed changes. Once again, we extend our heartfelt gratitude for your dedication and passion in helping us shape the future of Scrumlr.
 
-Thank you 🙏 for looking into our contribution guide, and taking the time to improve scrumlr.io with your contribution.
+- [Prerequisites](#prerequisites)
+- [Development](#development)
+- [Testing](#testing)
+- [Translating](#translating)
+- [Definition of Done](#definition-of-done)
+- [Pull Request Guidelines](#pull-request-guidelines)
+- [Code Review](#code-review)
+- [Contact and Communication](#contact-and-communication)
+- [Code of Conduct](#code-of-conduct)
+- [License](#license)
 
-At the moment we are a small community, and we're open to discuss any of your ideas in order to establish some kind of
-standards here. Try to contribute to the best of our knowledge and belief, and we will treat you with respect and share
-our opinion with you.
+## Prerequisites
+Before you start contributing to our web application, please ensure that you have the following prerequisites set up on your development environment. On the front-end, you'll need to have **Node.js** installed, as well as **Yarn** (a fast and reliable package manager) to work with **React** and **TypeScript**. For the back-end, make sure you have **Go** installed, which will be used to build and run the back-end. Additionally, **Docker** is required to run the database locally. You can install Docker by following the official installation instructions for your operating system. Having these prerequisites in place will enable you to seamlessly set up and contribute to our web application. If you have any questions or need assistance with the setup, feel free to reach out for support.
 
-## Definition of done
+## Development
+1. Fork and clone the repository
+2. Create a branch for your pull request with `git checkout -b your-branch-name`
 
-Here's our definition done, which we agreed on.
+### Client
+1. Run `yarn` to install the Front-end dependencies
+2. Run `yarn start` to start the Front-end in development mode. 
+3. Open http://localhost:3000 to view it in the browser. The page will automatically reload if you make changes to the code. You will see the build errors and lint warnings in the console.
```

Comment:
maybe we could also put here that running on a local network is also possible using the `.env.development` file

Reply:
I think those three steps are sufficient to explain how to start the front-end locally.

Reply:
alright 👍